gdal/GDALmake.opt.in

```diff
@@ -58,16 +58,17 @@ OSX_VERSION_FRAMEWORK_PREFIX = ${OSX_FRAMEWORK_PREFIX}/Versions/@GDAL_VERSION_MA
 
 prefix		=	@prefix@
 exec_prefix	=	@exec_prefix@
+datarootdir	=	@datarootdir@
 INST_PREFIX	=	@exec_prefix@
 INST_INCLUDE	=	@includedir@
-INST_DATA 	=	@datadir@
+INST_DATA 	=	@datadir@/gdal
 INST_LIB	=	@libdir@
 INST_BIN	=	@bindir@
 INST_BASH_COMPLETION	=	@bashcompdir@
 INST_PYMOD      =       @pymoddir@
-INST_DOCS	=	@exec_prefix@/doc
+INST_DOCS	=	@htmldir@
```

Comment:
On my system, Ubuntu 16.04 (with a `./configure --without-libtool` build if that it is relevant), @htmldir@ gets substituted by ${docdir} in GDALmake.opt, but docdir is not defined, thus "make install-docs" results in the following error
```
/home/even/gdal/git/gdal/gdal/install-sh -d 
install:	no input file specified
```

Reply:
Thanks. I thought I had tested it but I can see now I hadn't actually executed this bit. I will fix it, maybe push a few times here to check with the CI and will post when I am happy.